src/Compilers/Core/Portable/Syntax/SyntaxList`1.cs

```diff
@@ -327,7 +327,7 @@ private static SyntaxList<TNode> CreateList(GreenNode creator, List<TNode> items
                 return default(SyntaxList<TNode>);
             }
 
-            var newGreen = creator.CreateList(items.Select(n => n.Green));
+            var newGreen = GreenNode.CreateList(items.Select(n => n.Green));
```

Comment:
It looks like `cretor` is also unused now. We should remove these unecessary parameters all the way up the tree.